src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -1386,10 +1381,10 @@ DropShardMoveSubscription(MultiConnection *connection, char *subscriptionName)
 
 
 /*
- * DropShardMoveUser drops the user with the given name if it exists.
+ * DropShardUser drops the user with the given name if it exists.
  */
-static void
-DropShardMoveUser(MultiConnection *connection, char *username)
+void
+DropShardUser(MultiConnection *connection, char *username)
```

Comment:
I think it makes sense to drop the Shard word in these functions now that you made them more generic:
```suggestion
DropUser(MultiConnection *connection, char *username)
```